scripts/build.js

```diff
@@ -0,0 +1,101 @@
+var fs = require('fs');
+var glob = require('glob');
+var path = require('path');
+var req = require('superagent-promise')(require('superagent'), Promise);
+var urlJoin = require('url-join');
+var yaml = require('js-yaml');
+
+var CDN = 'https://unpkg.com/';
+
+// Major versions.
+var AFRAME_VERSIONS = ['0.2.0', '0.3.0'];
```

Comment:
Well so many useless versions there I believe almost nobody is using now <0.2.0 right?